src/main/java/org/sagebionetworks/web/client/widget/entity/browse/FilesBrowserViewImpl.java

```diff
@@ -58,15 +61,13 @@ public FilesBrowserViewImpl(FilesBrowserViewImplUiBinder binder, EntityTreeBrows
 		});
 		entityTreeBrowser.setIsEmptyCallback(isEmpty -> {
 			if (isEmpty) {
-				downloadHelp.setHelpMarkdown("There are no downloadable items in this folder.");
-				downloadHelp.setVisible(true);
```

Comment:
Let's totally remove the downloadHelp widget from the Java (line 41) and the corresponding XML